atmos_model.F90

```diff
@@ -1977,6 +1977,58 @@ subroutine assign_importdata(jdat, rc)
             endif
           endif
 
+! get zonal ocean current:
+!--------------------------------------------------------------------------
+          fldname = 'ocn_current_zonal'
+          if (trim(impfield_name) == trim(fldname)) then
+            findex  = queryImportFields(fldname)
+            if (importFieldsValid(findex) .and. GFS_control%cplocn2atm) then
```

Comment:
To confirm....cplocn2atm is always true for the global configurations. When the ufs fields exchange is used for mom6 regional, these fields will now be available and connected for the global configuration. But they still will not be used, because the ``icplocn2atm`` can still be 0.  Is that correct?

Reply:
@DeniseWorthen The default value for icplocn2atm is 0. To use this parameter, it is necessary to add icplocn2atm = 1 in input.nml.